crawlers/mooncrawl/mooncrawl/queries_crawler/cli.py

```diff
@@ -0,0 +1,91 @@
+import argparse
+import logging
+
+from moonstreamdb.blockchain import AvailableBlockchainType
+
+from ..actions import push_data_to_bucket
+from ..settings import MOONSTREAM_S3_DATA_BUCKET, MOONSTREAM_S3_DATA_BUCKET_PREFIX
+from .actions import (
+    OutputType,
+    fetch_data_from_db,
+    fetch_query_from_journal,
+    prepare_output,
+    query_validation,
+)
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def parser_queries_execute_handler(args: argparse.Namespace) -> None:
```

Comment:
After discussing with @kompotkot and @Andrei-Dolgolev:

We realized that we want this crawler to go through the Query API like any other user would.

The plan is to remove this `mooncrawl/queries_crawler` code and update the Moonstream Python client to provide this functionality similarly to how we used it in `autocorns biologist`:
https://github.com/bugout-dev/autocorns/blob/cf00fb492de254821730a256d238d5a332810db6/autocorns/biologist.py#L371

We can remove the existing Moonstream Python client, bump the client version, and publish the new client.